hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceWALReader.java

```diff
@@ -270,7 +273,9 @@ public Path getCurrentPath() {
   private boolean checkBufferQuota() {
     // try not to go over total quota
     if (!this.getSourceManager().checkBufferQuota(this.source.getPeerId())) {
-      Threads.sleep(sleepForRetries);
+      LOG.warn("PeerId={}, sleep {}ms for source reader, current WAL is {}",
```

Comment:
Will this be too aggresive? We may log it every 300ms?

Reply:
It can be removed, since ReplicationSourceManager#checkBufferQuota has already WARN logs.